fbw-a380x/src/systems/instruments/src/PFD/FMA.tsx

```diff
@@ -69,6 +72,38 @@ export class FMA extends DisplayComponent<{ bus: EventBus; isAttExcessive: Subsc
 
   private AB3Message = Subject.create(false);
 
+  private isUnrestrictedClimbDescent: number = 0;
+
+  private readonly sub = this.props.bus.getSubscriber<PFDSimvars & Arinc429Values & SimplaneValues>();
+
+  private readonly selectedFpa = ConsumerSubject.create(this.sub.on('selectedFpa'), 0);
+
+  private readonly selectedVs = ConsumerSubject.create(this.sub.on('selectedVs'), 0);
+
+  private readonly selectedAltitude = ConsumerSubject.create(this.sub.on('selectedAltitude'), 0);
+
+  private readonly altitude = Arinc429ConsumerSubject.create(this.sub.on('altitudeAr'));
+
+  private readonly activeVerticalModeConsumer = ConsumerSubject.create(this.sub.on('activeVerticalMode'), 0);
+
+  private readonly UnrestrictedClimbDescent = MappedSubject.create(
```

Comment:
```suggestion
  private readonly unrestrictedClimbDescent = MappedSubject.create(
```

camelCase with lower case first letter please

Reply:
Of course, I don't know why I typed it like this 😅